### PR TITLE
Set the domains from the secrets file, default to application setting

### PR DESCRIPTION
**Pull request by bfcoder**
Mirror the user tool setup and set all the subdomains via the secrets.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -381,7 +381,7 @@
       lti_key: Application::COURSECOMPLETION,
       site_url: secrets.canvas_url,
       canvas_token: secrets.canvas_token,
-      domain: "#{Application::COURSECOMPLETION}.#{secrets.application_root_domain}",
+      domain: "#{secrets.course_completion_subdomain.presence || Application::COURSECOMPLETION}.#{secrets.application_root_domain}",
```
[the remark this answers is not in the document]
Comment (bfcoder) [reply db/seeds.rb]: It's ok in this case.

File: db/seeds.rb [hunk 2]
```diff
@@ -296,7 +296,7 @@
       lti_key: Application::SURVEYAGGREGATION,
       site_url: secrets.canvas_url,
       canvas_token: secrets.canvas_token,
-      domain: "#{Application::SURVEYAGGREGATION}.#{secrets.application_root_domain}",
+      domain: "#{secrets.survey_aggregation_subdomain.presence || Application::SURVEYAGGREGATION}.#{secrets.application_root_domain}",
```
[the remark this answers is not in the document]
Comment (bfcoder) [reply db/seeds.rb]: It's ok in this case.

File: db/seeds.rb [hunk 3]
```diff
@@ -256,7 +256,7 @@
       lti_key: Application::EXAMPROCTOR,
       site_url: secrets.canvas_url,
       canvas_token: secrets.canvas_token,
-      domain: "#{Application::EXAMPROCTOR}.#{secrets.application_root_domain}",
+      domain: "#{secrets.exam_proctor_subdomain.presence || Application::EXAMPROCTOR}.#{secrets.application_root_domain}",
```
[the remark this answers is not in the document]
Comment (bfcoder) [reply db/seeds.rb]: It's ok in this case.

File: db/seeds.rb [hunk 4]
```diff
@@ -74,7 +74,7 @@
     application_instances: [{
       lti_key: Application::ADMIN,
       site_url: secrets.canvas_url,
-      domain: "#{Application::ADMIN}.#{Rails.application.secrets.application_root_domain}",
+      domain: "#{secrets.admin_subdomain.presence || Application::ADMIN}.#{Rails.application.secrets.application_root_domain}",
```
[the remark this answers is not in the document]
Comment (bfcoder) [reply db/seeds.rb]: It's ok in this case.

**Comment by conradbeach** [comment]
These changes look good to me.

**Pull request merged**